src/vivarium_public_health/risks/distributions.py

```diff
@@ -137,7 +137,7 @@ def ppf(self, x):
         if not np.allclose(1, np.sum(sorted_exposures, axis=1)):
             raise MissingDataError('All exposure data returned as 0.')
         exposure_sum = sorted_exposures.cumsum(axis='columns')
-        category_index = (exposure_sum.T < x).T.sum('columns')
+        category_index = pd.concat([exposure_sum[c] < x for c in exposure_sum.columns], axis=1).sum(axis=1)
```

Comment:
Cool.  You could also use np.sum([...], axis=1) and dodge the concat overhead.  Though that's pretty trivial here compared to what we had before.

Reply:
Just tested it, and it's actually `np.sum([...], axis=0)`, but it also seems to take about 2x as much time as the `pd.concat` apporach.